uiTests/src/test/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala

```diff
@@ -94,6 +94,13 @@ class UITestSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_WITH
     // 5 branch rows (`develop` is no longer there) + 3 commit rows
     // (1 commit of `allow-ownership-link` and 3 commits of `call-ws` are all gone)
     Assert.assertEquals(8, branchAndCommitRowsCount)
+
+    intelliJ.machete.checkoutBranch("develop")
+    intelliJ.machete.slideOutBranch("call-ws")
+    intelliJ.machete.rejectBranchDeletionOnSlideOut()
```

Comment:
This was stalling on the old implementation since the dialog wasn't showing up at all.
It'd also fail on `acceptBranchDeletionOnSlideOut`,  &mdash; I only added `rejectBranchDeletionOnSlideOut` for the sake of _crop rotation_ aka _płodozmian_ 🌾